tock/api/views.py

```diff
@@ -171,6 +180,33 @@ def get_queryset(self):
             .filter(user_data__current_employee=True) \
             .order_by('last_name', 'first_name')
 
+class Submissions(generics.ListAPIView):
+    """
+    Returns a list of users and the number of timecards they have
+    submitted on time since the requested reporting period
+    """
+
+    serializer_class = SubmissionSerializer
+
+    def get_queryset(self):
+        rp_num = int(self.kwargs['num_past_reporting_periods'])
```

Comment:
A bit of boundary handling would be a great next step here, as-is this throws an exception if someone attempts to ask "How many timecards, since X timecards ago have been submitted on time" for values of X greater than we have reporting periods.

Reply:
Since switching to the `list()[-1]` suggestion, I believe there should no longer be any problems with bigger-than-reality requests